specs/basic_usage.spec.ts

```diff
@@ -97,8 +107,8 @@ test('layer vueI18n options properties are merge and override by priority', asyn
 
   await page.click(`#switch-locale-path-usages .switch-to-fr a`)
   await waitForURL(page, '/fr')
-  expect(await getText(page, '#snake-case')).toEqual('À-propos-de-ce-site')
-  expect(await getText(page, '#pascal-case')).toEqual('ÀProposDeCeSite')
+  expect(await getText(page, '#snake-case')).toEqual('Should-be-overridden')
+  expect(await getText(page, '#pascal-case')).toEqual('ShouldBeOverridden')
```

Comment:
```suggestion
  // Project VueI18n config overwrites `Should-be-overridden` and `ShouldBeOverridden`
  expect(await getText(page, '#snake-case')).toEqual('À-propos-de-ce-site')
  expect(await getText(page, '#pascal-case')).toEqual('ÀProposDeCeSite')
```

I know this is still WIP, but this should be reverted so otherwise you may be testing incorrect behavior 😅

Reply:
Thanks!
I've reverted these assetions.
Fixed in https://github.com/nuxt-modules/i18n/pull/2558/commits/01bb141b75468454d746220d0594c3d46bf15ed4